### PR TITLE
Raincatch 1255

### PR DESCRIPTION
**Pull request by jstaffor**
Please PLEASE, please render the file and read it before looking at the changes. You'll understand my approach in terms of the full repo - thanks :)

### CODE REVIEW

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 1]
```diff
@@ -1,11 +1,21 @@
-[id='pro-running-the-published-repositories-{chapter}']
-= Introduction
+[id='{context}-pro-running-the-demo-repositories']
+= Running the Demo App from the Published Repositories
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: `Running the {RC-NAME_Short} applications on local machine `

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+
+The {WFM-RC-NameShort} demo applications in this section are dependent on the _public NPM Registry_ and, do not require compilation via the TypeScript compiler and linking via Lerna.js.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: +The {WFM-RC-NameShort} applications

Comment (wtrocki) [reply docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: `do not require compilation via the TypeScript compiler and linking via Lerna.js.` - Let's remove this as this content is not needed and missleading for eventual reader.

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc (hunk 1 above)
Comment (jstaffor) [reply docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: I'm going to add a variable here so you can call them what you want when you/Eng/Product have made a decision on what to call them.

FYI - The original heading from Eng was "Running the Published Repositories for the Demo App".

File: docs/workforce-management-framework/upstream-1/running-the-published-repositories.adoc [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
-:chapter: running-the-published-repositories
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: running-the-published-repositories
+
+= Running the Demo App from the Published Repositories
 
-ifdef::upstream[]
-[discrete]
-endif::upstream[]
-= Running the Published Repositories for the Demo App
```
Comment (jstaffor) [inline docs/workforce-management-framework/upstream-1/running-the-published-repositories.adoc:8]: I'm going to add a variable here so you can call them what you want when you/Eng/Product have made a decision on what to call them.

File: docs/workforce-management-framework/topics/pro-running-the-demo-app.adoc [hunk 4]
```diff
@@ -1,12 +1,18 @@
-
 [id='{context}-pro-running-the-demo-app']
 = Running the Demo App
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/pro-running-the-demo-app.adoc]: I'm going to add a variable here so you can call them what you want when you/Eng/Product have made a decision on what to call them.

File: docs/shared/attributes.adoc [hunk 5]
```diff
@@ -16,6 +16,8 @@
 //***NEW START***
 :WFM-RC-NameLong: RainCatcher
 :WFM-RC-NameShort: RainCatcher
+:WFM-RC-AppType1: TYPE_1
```
Comment (jstaffor) [inline docs/shared/attributes.adoc:19]: @wtrocki - add whatever makes sense to this variable and the variable below before merging.

The variables have been placed throughout the docs so there will naturally be a cascading effect once they are changed.

Comment (wtrocki) [reply docs/shared/attributes.adoc:19]: I was thinking just to not call them demo as this implies some things like quality of the application etc. Just applications which is fine and do not need variable. 
We can make this improvement on master as well so no worries.

**Comment by wtrocki** [comment]
@jstaffor I'm ok to merge this.

**Comment by jstaffor** [comment]
:+1:

**Comment by wtrocki** [comment]
@jstaffor Some git problems. Looks like @paolobueno changes weren't included.

**Comment by paolobueno** [comment]
@wtrocki now I see that this PR targetted the feature branch as opposed to master, going to investigate and try to recover.

**Pull request merged**